README.md

```diff
@@ -470,8 +470,8 @@ SLF4J: Defaulting to no-operation (NOP) logger implementation
 SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.
 ```
 
-JDA currently provides a fallback Logger in case that no SLF4J implementation is present.
-We strongly recommend to use one though, as that can improve speed and allows you to customize the Logger as well as log to files
+JDA provides a fallback Logger in the case that no SLF4J implementation is present.
+We strongly recommend to use one though, as that can improve speed and allows you to customize the Logger as well as route your Logger's output to a file.
```

Comment:
This seems like a more confusing wording, I don't get why this was changed

Reply:
Changed because the word currently was redundant as it's likely not gonna get removed, and the final bit was to make it clearer (though it wasn't necessary)

Reply:
Doesn't seem like a useful change to me and the last part makes it more confusing than anything